### ISSUE TITLE
Agent and Server Ansible roles do not remove yum cache folders, and possible redundancy with cleaning yum cache

### ISSUE THREAD

**Issue by cep235** [distributed-system-analysis/pbench#1706]
This is more of a nitpick and I'll be happy to submit a PR if needed.. I'm running RHEL 8.2. At least for my version and release of RHEL ([and apparently Fedora as well](https://docs.fedoraproject.org/en-US/Fedora_Core/5/html/Software_Management_Guide/sn-yum-caching.html)), the `pbench-clean-yum-cache` roles under `agent` and under `server` do not remove the cache folders for yum on my machine.

For example, the yaml file for `.../agent/ansible/roles/pbench-clean-yum-cache` looks like this

```yaml
---
- name: Clean yum cache
  command: yum clean all

- name: Delete /var/yum/cache directory
  file:
    path: /var/yum/cache
    state: absent
```

Here is what happens after I run the role:

```bash
[ec2-user@<redacted> ~]$ cat /etc/redhat-release 
Red Hat Enterprise Linux release 8.2 (Ootpa)
[ec2-user@<redacted> ~]$ ls -la /var | grep yum
[ec2-user@<redacted> ~]$ ls -la /var/cache | grep yum
drwxr-xr-x.  2 root root    6 Apr 23 05:19 yum
```

At least on RHEL 8.2 and Fedora, the `yum` cache directory is `/var/cache/yum`, rather than `/var/yum/cache`. So basically, a non-existent folder is being deleted on my RHEL 8.2 machine and the `yum` cache folder remains. I'm not an expert with RHEL or Fedora, so this is a genuine question and I could be 100% wrong, but doesn't `rm -rf /var/cache/yum*` do effectively the same thing as `yum clean all` in your case, except for the fact `rm -rf` also removes the cache folder? So I wonder if you only need to do the following (unless you're also considering people who may have modified their cache path in `/etc/yum.conf`):

```yaml
---
- name: Delete all yum cache
  file:
    path: /var/cache/yum
    state: absent
```

  - Full url path to the `agent`'s role: https://github.com/distributed-system-analysis/pbench/blob/master/agent/ansible/roles/pbench-clean-yum-cache/tasks/main.yml#L7
  - Full url path to the `server`'s role: https://github.com/distributed-system-analysis/pbench/blob/master/server/ansible/roles/pbench-clean-yum-cache/tasks/main.yml#L7

**Comment by ndokos** [comment]
I'll push another commit to PR #1705 to fix this.

**Comment by portante** [comment]
Fixed by #1705.
